### ISSUE TITLE
Having 'scope' parameter in the authorization request seems to cause login failures

### ISSUE THREAD

**Issue by mefarazath** [aaronpk/sign-in-with-apple-example#1]
https://github.com/aaronpk/sign-in-with-apple-example/blob/924c92cbe6b4edc7d0801f90d475ac9f9f5e5b9c/index.php#L79

After I got rid of the scope query param from the authorize URL login works as expected

**Comment by aaronpk** [comment]
Thank you! You helped me track down a deeper problem!

It turns out that you're right, having `scope` in the request causes Apple to return a 500 server error when confirming the login on their site before it redirects back to the app.

**However**, the really weird part is that Apple apparently completely ignores the `scope` parameter the second time you log in to an app, so there is no error.

The very first time I logged in to an app while testing this code I didn't include the `scope` parameter, so it worked. Then I added the parameter to see if I could get it to return an email address, and it didn't. It also didn't fail, because I had already logged in once.

I was having trouble logging in with a new App ID I created, and this is the reason! I just tried removing the scope from my attempt and now I'm able to log in with new App IDs.

This is very inconsistent behavior by Apple, so I hope they fix it later.
